adhocteam/Head-Start-TTADP
# Add sorting and pagination to the main reports table

**Description of change**
Adds sorting and pagination to the main reports table. The sorting and pagination is implemented with the help of the backend. The number of reports per page in the activity reports table is set to 10.

**How to test**
- pull changes
- run the app and create reports (more than 10)
- note that the pagination is now part of the activity reports table

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/187
* https://github.com/HHS/Head-Start-TTADP/issues/273

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
